examples/package.json

```diff
@@ -1,74 +1,63 @@
 {
   "name": "@cognite/reveal-examples",
-  "version": "0.1.0",
   "description": "Reveal examples",
-  "main": "dist/index.js",
-  "types": "dist/src/index.d.ts",
-  "files": [
-    "dist"
-  ],
+  "version": "0.1.0",
+  "private": true,
   "scripts": {
-    "release": "webpack",
-    "build": "webpack --env.development",
-    "clean": "rimraf -rf dist/ target/",
-    "serve": "webpack-dev-server --env.development --debug --color --progress --watch",
-    "test": "jest",
-    "coverage": "jest --coverage",
-    "lint": "tslint --project ."
+    "start": "react-app-rewired start",
+    "build": "react-app-rewired build",
+    "test": "react-app-rewired test",
+    "eject": "react-scripts eject"
   },
   "author": "Svenn-Arne Dragly <dragly@cognite.com>",
   "contributors": [
     "Lars Moastuen <lars.moastuen@cognite.com>",
```

Comment:
Just noticed, Christopher and Maksim, you guys might want to add yourself to the list